recipe/meta.yaml

```diff
@@ -25,14 +25,15 @@ requirements:
     - traitlets >=4.3.1,<6.0.0
     - nbformat >=4.2.0
     - widgetsnbextension >=3.5.0,<3.6.0
+    - jupyterlab_widgets >=1.0.0
```

Comment:
Does this need to be a `host` requirement as well? Wondering about the `pip` resolver issue on CI. It may be the answer is no and `pip` is causing issues, but wanted to check 🙂

Reply:
Thanks for checking. We just released jupyterlab_widgets 1.0, and I can't install it yet (caching issues?), so I thought that was what the problem was.

Reply:
Yeah I think you're right. Missed that. In the log it says, "Solving environment: ...working... failed". May need to wait a bit and restart CI.